scripts/bundleTs.mjs

```diff
@@ -167,7 +167,7 @@ const excludedPackages = new Set(['@jest/globals']);
       let definitionFile = await fs.promises.readFile(filepath, 'utf8');
 
       rimraf.sync(path.resolve(packageDir, 'build/**/*.d.ts'));
-      fs.rmSync(path.resolve(packageDir, 'dist/'), {
+      fs.rmSync(path.resolve(packageDir, 'dist/**'), {
```

Comment:
this doesn't seem right - `rmSync` doesn't support globs, so it would just not do anything. It's some other `read` that fails.

I'll try to dig up a windows computer and reproduce

Reply:
Sure!

Reply:
Unfortunately the only windows machine I have access to is Windows 7, and the latest version of Node it's possible to install on it is v12 (which doesn't have `fs.rm` API), so I'm unable to debug this 😅 

@G-Rath you wouldn't have som spare time to at this, would you? 😀

Reply:
Yeah I can probably have a look tomorrow

Reply:
sweet, thanks!